app/validate_format.py

```diff
@@ -0,0 +1,43 @@
+from flask_restful import fields
+import string
+
+from . import db
+
+itemformat = {"id": fields.Integer,
+              "name": fields.String,
```

Comment:
Consider separating this part into a possible `serializers.py` module. For separation of concerns